config/crd/bases/ndb.nutanix.com_databases.yaml

```diff
@@ -109,83 +104,46 @@ spec:
                         type: object
                     type: object
                   size:
-                    default: 10
-                    description: Size of the database instance
-                    minimum: 10
+                    description: Size of the database instance, default 10, minimum
+                      10
```

Comment:
is this description accurate? are we still defaulting to 10?

Reply:
Good catch, no I removed the defaulter logic and retained the original validation logic for the minimum 10 GBs value. Fixed it in this PR itself.